code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -300,6 +300,61 @@
 	. = ..()
 	AddComponent(/datum/component/armor_plate)
 
+
+//Nanotrasen medical Protosuit
+
+/obj/item/clothing/suit/space/hardsuit/protosuit
+	name = "protosuit hardsuit"
+	desc = "A failed NanoTrasen experiment, intended to be a unique medical hardsuit based on stolen Syndicate samples."
```

Comment:
not spelled NanoTrasen
also this is a magic tendril item, why nanotrasen lore?

Reply:
same reason as NT brand disks

Reply:
Also it IS spelled NanoTrasen, just searched in-code

Reply:
no its Nanotrasen